src/max-width.js

```diff
@@ -0,0 +1,27 @@
+const { get, is, px, num, breaks, dec, media, merge } = require('./util')
+const { maxWidths } = require('./constants')
+const width = require('./width')
+
+module.exports = props => {
```

Comment:
Might be missing something, but is there a reason not to use the `responsiveStyle` utility for this?

Reply:
That's what I do - https://github.com/johnpolacek/styled-system-html/blob/ecfc999347102adabe81dca43e7d5271d29ce66a/src/styles.js#L7 - works great :)